app/models/adjustment.rb

```diff
@@ -39,8 +39,6 @@ def self.storage_locations_adjusted_for(organization)
   def split_difference
     pre_adjustment = line_items.partition { |line_item| line_item.quantity.positive? }
     increasing_adjustment, decreasing_adjustment = pre_adjustment.map { |adjustment| Adjustment.new(line_items: adjustment) }
-
-    decreasing_adjustment.line_items.each { |line_item| line_item.quantity *= -1 }
```

Comment:
I'm not sure why you remove this -- it is part of some subtle behavior that callers use to divvy into add-inventory and remove-inventory adjustments, but it assumes that they are all positive. I think this is why some of the tests are failing. Not that I think this is the most straightforward way to do things, but rather the callers of `split_difference` expect this behavior.